migrations/versions/f7cb1b951d51_message.py

```diff
@@ -0,0 +1,28 @@
+"""message
```

Comment:
Can we remove this file? We're currently not using alembic and shouldn't be running migrations. I think there is a note in the README, but if it is not clear I'd be happy to get a PR for it.

Reply:
ok Ill post a quick fix, The contrib docs are a bit confusing on this matter, maybe a little format or punctuation there could help. but Ill get rid of the migration file.